stetho/src/main/java/com/facebook/stetho/inspector/elements/AbstractChainedDescriptor.java

```diff
@@ -144,4 +144,14 @@ public final void setAttributesAsText(Object element, String text) {
   protected void onSetAttributesAsText(E element, String text) {
     mSuper.setAttributesAsText(element, text);
   }
+
+  @Override
+  @SuppressWarnings("unchecked")
+  public final void getStyleProperties(Object element, StyleAccumulator accumulator) {
```

Comment:
Now that I think about it, using "style" and "property" together is bad terminology. We already have something in the UI called "properties." I think we should just call this `getStyles`. We shouldn't throw in words like property, info, etc.